assets/kube-mixins/additional-prometheus-rule.yaml

```diff
@@ -0,0 +1,330 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
+    app.kubernetes.io/part-of: openshift-monitoring
+    prometheus: k8s
+    role: alert-rules
+  name: etcd-rules
```

Comment:
isn't it confusing that it's named "etcd-rules"? it's also clashing with the next file.

Reply:
oh yeah, copy-paste error :facepalm: